specification/dashboard/resource-manager/readme.go.md

```diff
@@ -0,0 +1,26 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+```yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: dashboard
+  clear-output-folder: true
+```
+
+### Go multi-api
+
+``` yaml $(go) && $(multiapi)
+batch:
+  - tag: package-2021-09-01-preview
+```
+
+### Tag: package-2021-09-01-preview and go
+
+These settings apply only when `--tag=package-2021-09-01-preview --go` is specified on the command line.
+Please also specify `--go-sdks-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+```yaml $(tag) == 'package-2021-09-01-preview' && $(go)
+output-folder: $(go-sdks-folder)/services/preview/$(namespace)/mgmt/2021-09-01-preview/$(namespace)
+```
```

Comment:
~~~suggestion
## Go

These settings apply only when `--go` is specified on the command line.

```yaml $(go) && $(track2)
azure-arm: true
license-header: MICROSOFT_MIT_NO_VERSION
module-name: sdk/resourcemanager/dashboard/armdashboard
module: github.com/Azure/azure-sdk-for-go/$(module-name)
output-folder: $(go-sdk-folder)/$(module-name)
```
~~~